exercises/gigasecond/gigasecond_test.go

```diff
@@ -53,7 +53,7 @@ func parse(s string, t *testing.T) time.Time {
 		} else {
 			t.Log(err)
 			t.Skip("(Not your problem.  " +
-				"please file issue at https://github.com/exercism/xgo.)")
+				"please file issue at https://github.com/exercism/go.)")
```

Comment:
Hm. Right. Maybe something like "This is unexpected. Consider posting an issue to..."